store/store.go

```diff
@@ -191,10 +212,12 @@ func generateMongoQuery(p *Params) bson.M {
 	//if optional parameters are present, then add them to the query
 	if len(p.Types) > 0 && p.Types[0] != "" {
 		groupDataQuery["type"] = bson.M{"$in": p.Types}
+		skipParamsQuery = true
```

Comment:
For some reason, we fetch 2x the pumpSettings.
One explicit by blip query, the second time when fetching all the rest of the data:
````
$or: [
	{
		"_active":true, 
		"_schemaVersion": {$gte:1, $lte:99}, 
		"_userId": "d3ae3cbe4f", 
		"subType": {$ne: "deviceParameter"}, 
		"time": {$lte: "2020-03-07T11:00:58.708Z"}
	},{
		"_active":true, 
		"_schemaVersion": {$gte:1, $lte:99}, 
		"_userId": "d3ae3cbe4f", 
		"level": {$in: [0, 1, 2, 3]}, 
		"subType": "deviceParameter", 
		"time": {$lte: "2020-03-07T11:00:58.708Z"}, 
		"type":"deviceEvent"
	}
]
````